### PR TITLE
feat: [CET-1507]: CET events chart icons + icons package version bump

### PR DESCRIPTION
**Pull request by guycastel**
'explode' icon:
image
'add-graph' icon:
image
'remove-graph' icon:
image
(See better in "Files changed").

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Review by abhinavrastogi-harness — DISMISSED** [review]

File: packages/icons/src/add-graph.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```
Comment (Sagar-Rajput7) [inline packages/icons/src/add-graph.svg:1]: are these optimized via svgo?

File: packages/icons/src/explode.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```
Comment (Sagar-Rajput7) [inline packages/icons/src/explode.svg:1]: same here

**Review by sunnykesh-harness — COMMENTED** [review]
Have we minimised the SVGs using SVGO or any similar tool?

File: packages/icons/src/remove-graph.svg [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```
Comment (Sagar-Rajput7) [inline packages/icons/src/remove-graph.svg:1]: same here

**Review by sunnykesh-harness — APPROVED** [review]

**Review by Sagar-Rajput7 — APPROVED** [review]

**Pull request merged**